src/test/RecipesView.test.js

```diff
@@ -0,0 +1,126 @@
+import axios from 'axios'
+import React from 'react'
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import RecipesView from '../RecipesView'
+import { BrowserRouter } from 'react-router-dom'
+
+jest.mock('axios')
+
+const recipesMockResponse = {
+    data: [
+        {
+            name: 'recipe a',
+            description: 'recipe a',
+            ingredients: [{ name: 'flour' }],
+        },
+        {
+            name: 'recipe b',
+            description: 'recipe b',
+            ingredients: [{ name: 'wheat' }],
+        },
+    ],
+}
+
+describe('RecipesView', () => {
+    afterEach(() => {
+        jest.clearAllMocks()
+    })
```

Comment:
two things
instead of `clearMocks` it is better to [reset them](https://jestjs.io/docs/jest-object#jestresetallmocks), the second one on top of removing previous usages, remove the implementation you provide to the mock, and then the tests are more isolated.
reseting the mocks is a good practice that be default must be added to every test suite, a way to do that is adding this piece of code inside a file called **setupTests** that is used to add shared configuration, check it for [extranet-portal](https://github.com/travelperk/frontend/blob/master/apps/extranet-portal/src/setupTests.js)